dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -127,7 +127,7 @@ uiConfig:
 
   orcidAuthUrl: https://sandbox.orcid.org/oauth/authorize
   orcidRedirectPath: /auth/orcid.org
-  orcidScope: /authorize
+  orcidScope: /authenticate
```

Comment:
odd...where is this dockstore.yml used? i was supposed to change it to the other scope.  the one mentioned in partialDockstore.yml

Reply:
It's linked in the '[setting up dockstore locally](https://wiki.oicr.on.ca/pages/viewpage.action?spaceKey=DOC&title=Setting+Up+Dockstore+Locally)' wiki page in [this](https://wiki.oicr.on.ca/pages/viewpage.action?spaceKey=DOC&title=Setting+Up+Dockstore+Locally#SettingUpDockstoreLocally-Dockstore.yml) paragraph, which is where I downloaded it.

Reply:
hmm...odd, well change it to the one in partialDockstore.yml and hope for the best